src/peft/tuners/vera/layer.py

```diff
@@ -100,6 +100,23 @@ def update_layer(
             # we can take any of the existing adapter's parameters, as they should all be identical
             vera_A_param = list(self.vera_A.values())[0]
             vera_B_param = list(self.vera_B.values())[0]
+
+            error_tmpl = (
+                "{} has a size of {} but {} is or greater required; this probably happened because an additional VeRA "
```

Comment:
a typo - should be "{} has a size of {} but {} or greater is required [...]", right?

Reply:
we could also have separate messages for incompatible r dimension or input/output dimensions;
can also give a hint that adding adapters in different order *may* resolve the problem